trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/GravitinoConfig.java

```diff
@@ -140,4 +159,50 @@ static class ConfigEntry {
       CONFIG_DEFINITIONS.put(key, this);
     }
   }
+
+  static class TrinoConfig {
+
+    private final Properties properties;
+
+    public TrinoConfig() {
+      this.properties = new Properties();
+      try {
+        String configFileName = System.getProperty("config");
+        properties.load(new FileInputStream(configFileName));
+        if (properties.containsKey(TRINO_CATALOG_STORE)
+            && !properties
+                .getProperty(TRINO_CATALOG_STORE)
+                .equals(TRINO_CATALOG_STORE_DEFAULT_VALUE)) {
```

Comment:
So what the value of `TRINO_CATALOG_STORE` is `File`? the final result is true or false here?

Reply:
Maybe `file` or `memory`. if it's file return true

Reply:
I mean whether it is case sensitive such as `file` or `file`.

Reply:
It's false, it's case insensitive